app/code/Magento/InventoryConfiguration/composer.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "magento/module-inventory-configuration",
+    "description": "N/A",
+    "require": {
+        "php": "7.0.2|7.0.4|~7.0.6|~7.1.0",
+        "magento/framework": "100.2.*",
+        "magento/module-backend": "100.2.*",
+        "magento/module-ui": "100.2.*"
```

Comment:
Looks like we already have dependency on `Catalog`
But we do not have dependency on `Backend`

Also need to update global `composer.json` (located in root folder) and `composer.lock` (run composer update)
Pay attention that PHP version on env must be not bigger than 7.0.*